addons/mail/static/src/core/web/message_patch.js

```diff
@@ -64,57 +51,4 @@ patch(Message.prototype, {
     openRecord() {
         this.message.thread.open();
     },
-
-    /**
-     * @returns {string}
-     */
-    formatTracking(trackingType, trackingValue) {
-        switch (trackingType) {
-            case "boolean":
-                return trackingValue.value ? _t("Yes") : _t("No");
-            /**
-             * many2one formatter exists but is expecting id/display_name or data
-             * object but only the target record name is known in this context.
-             *
-             * Selection formatter exists but requires knowing all
-             * possibilities and they are not given in this context.
-             */
-            case "char":
-            case "many2one":
-            case "selection":
-                return formatChar(trackingValue.value);
-            case "date": {
-                const value = trackingValue.value
-                    ? deserializeDate(trackingValue.value)
-                    : trackingValue.value;
-                return formatDate(value);
-            }
-            case "datetime": {
-                const value = trackingValue.value
-                    ? deserializeDateTime(trackingValue.value)
-                    : trackingValue.value;
-                return formatDateTime(value);
-            }
-            case "float":
-                return formatFloat(trackingValue.value);
-            case "integer":
-                return formatInteger(trackingValue.value);
-            case "text":
-                return formatText(trackingValue.value);
-            case "monetary":
-                return formatMonetary(trackingValue.value, {
-                    currencyId: trackingValue.currencyId,
-                });
-            default:
-                return trackingValue.value;
-        }
-    },
-
-    /**
-     * @returns {string}
-     */
-    formatTrackingOrNone(trackingType, trackingValue) {
-        const formattedValue = this.formatTracking(trackingType, trackingValue);
-        return formattedValue || _t("None");
-    },
```

Comment:
The method should be kept in component to respect stable policy. It's ok to have them call the code in model, but we should not remove them unless we are doing `master` changes.